scripts/install_fonts.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
+set -o errexit \
+    -o pipefail \
+    -o nounset
+
+mkdir /fonts
+
+# Spartan
+for f in /usr/share/fonts/opentype/league-spartan/LeagueSpartan-*.otf; do
+    _font_file_name="$(basename "${f}")"
+    _new_font_name="$(sed 's/League//' <<< "${_font_file_name}")"
+    cp -v "${f}" "/fonts/${_new_font_name}"
+done
+
+cp -v /fonts/Spartan-Regular.otf /fonts/Spartan.otf
+
+# Jost
+curl -fL --output /tmp/jost.zip https://fonts.google.com/download?family=Jost
```

Comment:
It could be cleaner to install all fonts within a subfolder of the main directory to prevent clutter. This may require altering TechCheck.cls such that it can still locate the fonts.

Reply:
For some reason I thought fixing this would be much harder than it ended up being. >_<